packages/inferno-compat/src/index.ts

```diff
@@ -132,11 +132,26 @@ function normalizeGenericProps(props) {
 }
 
 function normalizeFormProps<P>(name: string, props: Props<P> | any) {
+  const validLineInputs = {
```

Comment:
As mentioned earlier lets hoist this object outside function so its not created everytime normalizeProps is called. Otherwise LGTm

Reply:
Its static information anyway

Reply:
My bad, thought I'd sorted that - have updated.